pkgs/servers/web-apps/freshrss/extensions/default.nix

```
@@ -0,0 +1,137 @@
+{ config
+, lib
+, fetchFromGitHub
+, fetchFromGitLab
+, freshrss-utils
+}:
+
+let
+  inherit (freshrss-utils) buildFreshRssExtension;
```

I'm thinking that it can be imported here instead.
```suggestion
  buildFreshRssExtension = (callPackage ./freshrss-utils.nix {}).buildFreshRssExtension;
```